docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<p>robsut --&gt; robust</p><blockquote>What we can see in this parallel axis plot is that there is a clear tradeoff between robustly high reliability and robsut low maximum polution</blockquote><p>Add how this can be seen (probably diagonal lines, or high density of lines on opposites of the Y-axis between variables)</p><p> </p><p>Aside from only plotting, can we also express these metrics numerically?</p><p> </p><p>Finally, a summary of what is and can be done in this whole notebook needs to be added, with some links and resources where to go further.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<ol><li>there is a dataframe so yes we have numbers</li><li>no idea what you have in mind and whether it belongs in the documentation.</li></ol><p> </p>

Reply:
<p>Let’s discuss this in person</p>